src/user/index.ts

```diff
@@ -232,8 +280,11 @@ export const getUserActivationState = async (uuid: string) => {
 			...(await Oauth.getAuthHeaders())
 		}
 	})
-	if(result.status !== 200) {
-		throw new AuthError("Unable to get authentication token");
+	if(result.status === 403) {
+		throw new AuthError("You do not have access to view a user's activation status. (403 Forbidden)");
+	}
+	else if(result.status !== 200) {
+		throw new AuthError("We were unable to get this user's activation status.");
```

Comment:
Why auth error?

Reply:
Changed to `ApIGetError` instead because it has nothing to do with Auth